### ISSUE TITLE
Add robots.txt to apprtc

### ISSUE THREAD

**Issue by juberti** [webrtc/samples#438]
Right now we are getting many robots.txt requests per day.
We should exclude everything except the root directory (e.g. index.html)

**Comment by samdutton** [comment]
Is this right, from the apprtc directory?

User-agent: *
Disallow: /
Allow: /index.html


We might want to use the Webmaster Tools robots.txt tester
<https://support.google.com/webmasters/answer/6062598>.

**Comment by juberti** [comment]
Does that allow requests to /?
I was thinking Disallow: /r/, Allow: /

**Comment by samdutton** [comment]
My example is meant to disallow everything except /index.html.

Do we want anything except /index.html to get crawled?

**Comment by juberti** [comment]
My question was whether / is treated the same as /index.html. If yes, then that is fine.

**Comment by samdutton** [comment]
Disallow: /

means disallow everything (doesn't it?)

**Comment by juberti** [comment]
I wasn't sure if there was some way to allow just the root but nothing underneath. But I checked, and AFAICT, no dice.

Let's go with your suggestion, and we can add additional Allow: lines if needed in the future (e.g. some sort of about.html)

**Comment by samdutton** [comment]
If no objections, I think we should run the Webmaster Tools robots.txt
tester <https://support.google.com/webmasters/answer/6062598>.

This requires adding an HTML verification file first:
apprtc.appspot.com/google1b7eb21c5b594ba0.html.

**Comment by juberti** [comment]
ok

**Comment by samdutton** [comment]
See PR 439 <https://github.com/GoogleChrome/webrtc/pull/439>.

**Comment by chuckhays** [comment]
@samdutton Are you working on this or should I pick it up?

**Comment by samdutton** [comment]
apprtc.appspot.com has now been verified by Webmaster Tools.

I did some more research on how to exclude everything except root. This
(from this answer on Stack Overflow
<http://stackoverflow.com/questions/7226432/robots-txt-allow-root-only-disallow-everything-else>)
seems like the best option I've seen so far:

User-agent: *
Allow: /$
Disallow: /

I tried this out with the Webmaster Tools robots.txt tester and have pushed
it (along with handler added to app.yaml). This is now part of #66.

Vikas - this will need to be deployed.

Sam
